src/views/EventPage.vue

```diff
@@ -30,36 +34,6 @@ export default {
 
 <style lang="scss" scoped>
 .page-container {
-  background-image: url("../assets/sample-img.png");
-  height: 100vh;
-  width: 100%;
-
-  .nav-btns {
-    position: fixed;
-    top: 2%;
-    width: 80%;
-    height: 10%;
-    display: flex;
-    flex-direction: row;
-    justify-content: space-between;
-    align-content: center;
-  }
-
-  .title-container {
-    position: absolute;
-    top: 40%;
-    width: 80%;
-    left: 10%;
-    display: flex;
-    flex-direction: row;
-    justify-content: space-between;
-    align-content: center;
-
-    .event-title {
-      font-family: "Josefin Sans", sans-serif;
-      font-size: 2.2em;
-      letter-spacing: -0.05em;
-    }
-  }
+  @include page-container-middle;
```

Comment:
netlify logs spot problem here

Reply:
oops yea figured it out i forgot we havent merged mixins

Reply:
should always use master as base - simply commenting it out is gonna absolutely merge conflict the heck out of it + break styles for this branch which I feel is not recomm

Reply:
yea i used master as base but i thought mixin was merged so i did this